utils/update_doc_version.sh

```diff
@@ -4,7 +4,7 @@
 # Example: Suppose that before running this script, the build.gradle reads 7.8.9.  This script will replace 
 # all occurrences of #.#.# with 7.8.9 in the README files.
 
-# Get the current maven project version.
+# Get the current project version.
 RELEASED_VERSION=$(./gradlew properties | grep -o 'version.*' | sed -n -e 's/^.*version: //p')
```

Comment:
Maybe that command line should be split into a separate script to be used in all the places it's needed?  Looks like you changed it in update_gradle_version.sh but not here :)  (is the same thing elsewhere?)